### PR TITLE
chore(docs): Some SQL queries to copy paste for advanced debugging

### PR DESCRIPTION
**Pull request by danieldietzler**

### CODE REVIEW

File: docs/docs/guides/database-queries.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+should be replaced by the according value [set in your `.env` file](https://docs.immich.app/docs/install/environment-variables#database).
+
+### GUI (Portainer, Unraid, ...)
+
+When using a GUI, it will most likely provide the ability to create a shell within the container.
+In that shell, simply run `psql immich DB_USERNAME` while replacing your `DB_USERNAME` like explained above.
+
+### Kubernetes
+
+If you're using Kubernetes in your home lab you are crazy and know how to run a command anyway.
```
Comment (samip5) [inline docs/docs/guides/database-queries.mdx:27]: Nit: I don't know if that should call people crazy. ;)

Comment (danieldietzler) [reply docs/docs/guides/database-queries.mdx:27]: Thanks for the review!
That's why it's still a draft :)

**Comment by danieldietzler** [comment]
> This should probably come with a big warning that mucking around in the database might set the moon on fire etc etc so have good backups ;)

I will most certainly not include any modifying queries. :D
But you're probably right and there is no harm in having a backup reminder here and there

**Comment by danieldietzler** [comment]
> mucking around in the database might set the moon on fire etc etc so have good backups ;)

May I quote this in the docs 🔥 ? haha

**Comment by bo0tzz** [comment]
Go ahead 😄

**Review by jrasm91 — APPROVED** [review]
TYVM

**Pull request merged**